### DISCUSSION TITLE
XDMF Time Series Compression

### DISCUSSION THREAD

**Discussion by masonajones** [nschloe/meshio#1245] [Q&A]
As far as I can tell, compression is not allowed in the XDMF time series writer for HDF5 files. Am I missing something? And if not, is there a reason it's not enabled?

**Comment by nschloe** [comment]
What makes you think it's not enabled?

**Reply by masonajones** [reply]
Compression and compression level are arguments for XdmfWriter, but not for TimeSeriesWriter. Similarly they're passed to h5_file in XdmfWriter but not in TimeSeriesWriter.
